### DISCUSSION TITLE
spacy-api-docker documentation is not readable

### DISCUSSION THREAD

**Discussion by sebastienheyd** [explosion/spaCy#12778] [Help: Other Questions]
Hello,

The documentation for [spacy-api-docker-v3](
https://spacy.io/universe/project/spacy-api-docker-v3) and [spacy-api-docker](https://spacy.io/universe/project/spacy-api-docker) is not accessible in my browser.

I'm using Ubuntu, and whether it's Firefox or Chrome, I get a black screen with the same error: "Application error: a client-side exception has occurred (see the browser console for more information)."

In the console, I see the following messages:

Error: The language "docker" has no grammar.
NextJS 33
framework-3b5a00d5d7e8d93b.js:9:69816

Error: The language "docker" has no grammar.
NextJS 33
main-a0f603ce323043fd.js:1:16311

A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred main-a0f603ce323043fd.js:1:16328

**Comment by adrianeboyd** [comment]
Thanks for the note, we are looking into it!

**Reply by adrianeboyd** [reply]
Should be fixed now, let us know if it's not working after a hard refresh on your end!

**Comment by sebastienheyd** [comment]
Yes ! Everything is ok now, thanks ! :+1:
